nerfstudio/models/nerfacto.py

```diff
@@ -290,12 +294,15 @@ def get_outputs(self, ray_bundle: RayBundle):
 
         rgb = self.renderer_rgb(rgb=field_outputs[FieldHeadNames.RGB], weights=weights)
         depth = self.renderer_depth(weights=weights, ray_samples=ray_samples)
+        with torch.no_grad():
+            depth_med = self.renderer_depth_med(weights=weights, ray_samples=ray_samples)
```

Comment:
It doesn't look like this is ever used

Reply:
I added this because I need average depths for the depth rank loss, but median depth is more useful to visualize. It's being added to the outputs, for visualization.

Reply:
We should probably change the behavior so that "depth" is median depth and "expected_depth" is the expected one which gradients go through, otherwise downstream code could break like pointcloud generation